### ISSUE TITLE
addChart use of "y_axis" minimum : 0 does not result in expected behavior

### ISSUE THREAD

**Issue by agadsby** [qax-os/excelize#1039]
According to the documentation if you want to pin the y_axis (or x_axis) to 0 you should use the attribute
     "y_axis" : "minimum", 0 
but that actually sets the axis is "auto" - this prevents pinning the y_axis to 0, which is probably the most common value.

The solution is to modify drawing.go function drawPlotAreaValAx and delete the test

```
//      if formatSet.YAxis.Minimum == 0 {       // GADSBY fixes incorrect application of AUTO if y_axis is 0
//              min = nil                                                                                              
//      }
```
this restores the expected behaviour in that auto is the default (starting) value for y_axis and setting it to 0 remove the default.

Example test program illustrates the problem in that the baseline is set to 790,000 not 0 which is what would be expected.

```
package main

import (
    "fmt"

    "github.com/xuri/excelize/v2"
)

func main() {
    categories := map[string]string{
        "A2": "00:10", "A3": "00:20", "A4": "00:30"}
    values := map[string]int{
        "B2": 800000, "B3": 810000, "B4": 815000 }
		
    f := excelize.NewFile()
    for k, v := range categories {
        f.SetCellValue("Sheet1", k, v)
    }
    for k, v := range values {
        f.SetCellValue("Sheet1", k, v)
    }
    if err := f.AddChart("Sheet1", "E1", `{
        "type": "areaStacked",
		"title" :	{"name" : "Show y_axis problem"},
		"y_axis":	{"minimum":0.0},
		"series":	[ {"name": "swap free", "values": "Sheet1!B2:B4", "categories" : "Sheet1!A2:A4"} ]
        
    }`); err != nil {
        fmt.Println(err)
        return
    }
    // Save spreadsheet file by the given path.
    if err := f.SaveAs("Book1.xlsx"); err != nil {
        fmt.Println(err)
    }
}
```

**Comment by agadsby** [comment]
Hi Xuri

I’ll do some further testing and also see if a similar change fixes the x_axis in the same way. Once I’ve done that I’ll do a pull. 

Andy

**Comment by xuri** [comment]
This issue has been fixed in v2.6.0. I will close this issue. If you have any questions, please let me know.
